Stratis.Bitcoin.Features.MemoryPool/MempoolBehavior.cs

```diff
@@ -465,7 +465,7 @@ private Task AttachedNode_MessageReceivedAsync(Node node, IncomingMessage messag
         private async Task SendAsTxInventory(Node node, IEnumerable<uint256> trxList)
         {
             this.logger.LogTrace("({0}:'{1}',{2}.{3}:{4})", nameof(node), node.RemoteSocketEndpoint, nameof(trxList), "trxList.Count", trxList?.Count());
-            Queue<InventoryVector> queue = new Queue<InventoryVector>(trxList.Select(s => new InventoryVector(InventoryType.MSG_TX, s)));
+            Queue<InventoryVector> queue = new Queue<InventoryVector>(trxList.Select(s => new InventoryVector(node.AddSupportedOptions(InventoryType.MSG_TX), s)));
```

Comment:
At this point we could make the Deployment class available to check if segwit is activated, for next PR I guess.